### DISCUSSION TITLE
no "create from image" option

### DISCUSSION THREAD

**Discussion by maromme** [mealie-recipes/mealie#4353] [Q&A] [answered]
Hi,
I'm starting into mealie and found that pr: https://github.com/mealie-recipes/mealie/pull/3974
There ist a button "import from image" shown. But not for me:

![grafik](https://github.com/user-attachments/assets/a86679ea-8aee-4978-885b-ccdd717b601c)

Im running mealie as an addon for homeassistant and set the openAI API key in my configuration:
```
ALLOW_SIGNUP: false
DATA_DIR: /config/addons_config/mealie_data
PGID: 1000
PUID: 1000
certfile: fullchain.pem
keyfile: privkey.pem
ssl: false
OPENAI_API_KEY: >-
  sk-bHan2xxxxxxxxxxxxxxxxxxxk7GcA
LOG_LEVEL: debug
```
I did try it with the "user API key" and the "project API key". Both no chance. And I got the "tier 1" approval 

What am I doing wrong? If I try uploading an image via "debug" -> "openai" I get the information "OpenAI not enabled"

Thanks a lot for your help!
Markus

**Comment by michael-genson** [comment]
Can you share your settings page? It should include what Mealie version you're on, and whether or not OpenAI is fully configured

**Comment by maromme** [comment]
I just configured zäthe api key. The rest I left with default values

![Screenshot_20241012_173523_Firefox](https://github.com/user-attachments/assets/86315540-1cdb-4a68-bc55-5e959399d88c)

**Reply by michael-genson** [reply, marked as answer]
![image](https://github.com/user-attachments/assets/832c9cf8-211e-4bcd-8639-ae8fd448d454)

It looks like your OpenAI API Key isn't set. I would double check your deployment and make sure the environment variable is actually getting set in the container

**Reply by maromme** [reply]
thank's a lot for your advice. I did try to set the OpenAI Values in the AddOn configuration:
![grafik](https://github.com/user-attachments/assets/8504faa2-43b1-443c-83a1-ebfed40f02bf)
though that didn't work out. 

but setting it in the config.yaml was doing the job
![grafik](https://github.com/user-attachments/assets/8d50d4a2-4575-45e0-8cc0-33bf1c146b51)

but for further confusions there is another config.yaml at
`/homeassistant/addons_config/db21ed7f-mealie/config.yaml`
Setting the OpenAI values there is not doing anything.

Anyway it's working now. Thanks for the help!
